src/odemis/acq/test/path_test.py

```diff
@@ -978,21 +1002,246 @@ def test_set_path(self):
         self.assertEqual(self.spec_det_sel.position.value, {'rx': 0})
         self.assertAlmostEqual(self.spec_sel.position.value["x"], 0.022)
 
+        # Setting fiber-align with an explicit target
+        self.optmngr.setPath("fiber-align", self.spec).result()
```

Comment:
I think it's cleaner to write `self.optmngr.setPath("fiber-align", detector=self.spec).result()`, also in the other places where you pass a detector.

Reply:
Updated the test case, but not the GUI, because it looks already quite explicit, with the "detector" keyword:
`main.opm.setPath(op_mode, self._speccnt_stream.detector)`.